src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_NUMBER = 10;
+    private K[] keys = (K[])new Object[MAX_ELEMENTS_NUMBER];
+    private V[] values = (V[]) new Object[MAX_ELEMENTS_NUMBER];
+    private int sizeOfStorage;
```

Comment:
```suggestion
    private int size;
```
for collections/storage type, size is understandable enough